applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -850,18 +852,21 @@ void audio_datapath_stream_out(const uint8_t *buf, size_t size, uint32_t sdu_ref
 	int ret;
 	size_t pcm_size;
 
-	ret = sw_codec_decode(buf, size, bad_frame, &ctrl_blk.decoded_data, &pcm_size);
+	ret = sw_codec_decode(buf, size, bad_frame, &ctrl_blk.decoded_data, &pcm_size, true);
```

Comment:
It seems the last parameter "lc3_playback_is_active" is always true.
We need to consider different scenarios, and also adding needed feature.
E.g., The API for start playing the stream, so we can choose when instead of playing the stream in the beginning.